spec/rest_in_peace/definition_proxy/resource_method_definitions_spec.rb

```diff
@@ -127,14 +127,14 @@
   context '#put' do
     it_behaves_like 'an instance method' do
       let(:http_verb) { :put }
-      let(:method_name) { :update }
+      let(:method_name) { :update_a }
```

Comment:
Because it was calling method `update` which is new one, here should calls just link to the path to put method.